### PR TITLE
man: document `--vacuum-time` default suffix is seconds

### PR DESCRIPTION
**Pull request by abitrolly**
# CODE FREEZE NOTICE

An -rc1 tag has been created and a release is being prepared, so please note that
PRs introducing new features and APIs will be held back until the new version
has been released.

### CODE REVIEW

File: man/journalctl.xml [hunk 1]
```diff
@@ -723,13 +723,12 @@
...
-        the specified size (specified with the usual <literal>K</literal>, <literal>M</literal>,
-        <literal>G</literal> and <literal>T</literal> suffixes), or all archived journal files contain no
-        data older than the specified timespan (specified with the usual <literal>s</literal>,
+        <listitem><para>Reduce journal files to the size specified with the usual <literal>K</literal>,
+        <literal>M</literal>, <literal>G</literal> and <literal>T</literal> suffixes, or all archived
+        journal files older than the timespan specified with the usual <literal>s</literal> (default),
         <literal>m</literal>, <literal>h</literal>, <literal>days</literal>, <literal>months</literal>,
-        <literal>weeks</literal> and <literal>years</literal> suffixes), or no more than the specified
-        number of separate journal files remain. Note that running <option>--vacuum-size=</option> has only
+        <literal>weeks</literal> and <literal>years</literal> suffixes, or leave only specified number
```
Comment (poettering) [inline man/journalctl.xml]: …leave only *the* specified number…

Comment (abitrolly) [reply man/journalctl.xml]: Thanks. Fixed.

File: man/journalctl.xml [hunk 2]
```diff
@@ -723,13 +723,12 @@
         <term><option>--vacuum-time=</option></term>
         <term><option>--vacuum-files=</option></term>
 
-        <listitem><para>Removes the oldest archived journal files until the disk space they use falls below
```
Comment (bluca) [inline man/journalctl.xml:741]: Why is the note about oldest archived journal being removed? Is this right @DaanDeMeyer ?

Comment (abitrolly) [reply man/journalctl.xml:741]: It is replaced with the phrase "Reduce journal files to the size specified " below.

Comment (bluca) [reply man/journalctl.xml:741]: Removing and reducing are not the same thing though. Which one is it?

Comment (abitrolly) [reply man/journalctl.xml:741]: `--vacuum-size` reduces size. `--vacuum-time` removes oldest entries.

Comment (abitrolly) [reply man/journalctl.xml:741]: @bluca I placed original description back and split the text for readability.

**Comment by abitrolly** [comment]
Is `--vacuum-size=0` a special case for something? https://github.com/systemd/systemd/issues/27401

If yes, it probably needs to be documented too.

**Comment by abitrolly** [comment]
I don't know why unit tests fail. Launchpad outage?

File: man/journalctl.xml [hunk 3]
```diff
@@ -738,13 +738,18 @@
...
-        <literal>weeks</literal> and <literal>years</literal> suffixes), or no more than the specified
-        number of separate journal files remain. Note that running <option>--vacuum-size=</option> has only
+        <listitem><para><option>--vacuum-size=</option> removes the oldest archived journal files until the
+        disk space they use falls below the size specified with <literal>K</literal>, <literal>M</literal>,
+        <literal>G</literal> and <literal>T</literal> suffixes.</para>
+
+        <para><option>--vacuum-time=</option> removes archived journal files older than the timespan
+        specified with <literal>s</literal> (default), <literal>m</literal>, <literal>h</literal>,
+        <literal>days</literal>, <literal>months</literal>, <literal>weeks</literal> and
+        <literal>years</literal> suffixes.
```
Comment (poettering) [inline man/journalctl.xml:748]: closing `</para>` missing here

**Comment by poettering** [comment]
> Is `--vacuum-size=0` a special case for something? #27401
> 
> If yes, it probably needs to be documented too.

It already is documented, i.e. one paragraph further down:

> --vacuum-size=, --vacuum-time= and --vacuum-files= may be combined in a single invocation to enforce any combination of a size, a time and a number of files limit on the archived journal files. Specifying any of these three parameters as zero is equivalent to not enforcing the specific limit, and is thus redundant.

**Comment by poettering** [comment]
To speed things up a bit I updated your commit a bit and posted it as #28019, left the attribution to you intact. Hope that's OK. Let's continue there.

**Pull request closed without merging**